config/300-route.yaml

```diff
@@ -28,3 +28,13 @@ spec:
     - knative
     - serving
   scope: Namespaced
+  additionalPrinterColumns:
+  - name: Domain
+    type: string
+    JSONPath: .status.domain
+  - name: Ready
+    type: boolean
+    JSONPath: .status.ready
+  - name: Age
+    type: date
+    JSONPath: .metadata.creationTimestamp
```

Comment:
I think that we probably want to do this across all of our CRDs and probably discuss the shape of what we want in an issue first?

Reply:
Sounds good, #1138 seems like a good place to discuss